recipes/cairo/meson/conanfile.py

```diff
@@ -224,166 +230,140 @@ def package(self):
             meson = self._configure_meson()
             meson.install()
         self._fix_library_names()
-        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
-        tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), "*.pdb")
+        files.rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        files.rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
 
     def package_info(self):
-        self.cpp_info.components["cairo_"].names["pkg_config"] = "cairo"
+        base_requirements = {"pixman::pixman"}
+        base_system_libs = {}
+
+        def add_component_and_base_requirements(component, requirements, system_libs=None):
+            self.cpp_info.components[component].set_property("pkg_config_name", component)
+            self.cpp_info.components[component].names["pkg_config"] = component
```

Comment:
Can be removed, `pkg_config` generator inherits from`pkg_config_name` property as well.

Reply:
I thought we broke the connection between v1 and v2 generators 🤔 was that only for CMake?

Since we have not removed it in other recipes, I dont see the harm in leaving it here as well